src/middlewares/auth.js

```diff
@@ -56,13 +56,6 @@ const checkToken = async (req, res, next) => {
     });
     return;
   }
-  if (!payload.isVerified) {
```

Comment:
Hi Kingsley, I can see you separated the functionality of checking for a token and if the user is verified. It would affect my implementation of the middleware in the GET request route. You can just add the checkToken middleware in the path as well. Cheers

Reply:
@kingsley-einstein has this being resolved? cc: @EziamakaNV